scripts/image

```diff
@@ -89,24 +89,17 @@ rm -rf $STAMPS_INSTALL
 mkdir -p $INSTALL
 
 # create baselayout
-mkdir -p $INSTALL/etc
-mkdir -p $INSTALL/dev
-mkdir -p $INSTALL/proc
-mkdir -p $INSTALL/run
-mkdir -p $INSTALL/sys
-mkdir -p $INSTALL/tmp
-mkdir -p $INSTALL/usr
-mkdir -p $INSTALL/var
-mkdir -p $INSTALL/flash
-mkdir -p $INSTALL/storage
+BASELAYOUT_DIRECTORIES="etc dev proc run sys tmp usr var flash storage"
+for directory in $BASELAYOUT_DIRECTORIES; do
+  mkdir -p $INSTALL/$directory
+done
 ln -sf /var/media $INSTALL/media
 ln -sf /usr/lib $INSTALL/lib
 ln -sf /usr/bin $INSTALL/bin
 ln -sf /usr/sbin $INSTALL/sbin
 
 if [ "$TARGET_ARCH" = "x86_64" -o "$TARGET_ARCH" = "powerpc64" ]; then
```

Comment:
Yes, we should remove `-o "$TARGET_ARCH" = "powerpc64"` as it is not supported.

Same here: https://github.com/LibreELEC/LibreELEC.tv/blob/652f675e7daef4f3ea93764ccfe1ca13517c80fb/packages/virtual/initramfs/package.mk#L27